### ISSUE TITLE
Sonification of arm doesn't work with keyboard

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#109]
When dragging the arm with the keyboard, sounds associated with  sonification do not play.

**Comment by jessegreenberg** [comment]
Commit above fixes issue, closing.
